lending/loan_management/doctype/loan/loan.json

```diff
@@ -482,12 +483,18 @@
    "fieldtype": "Link",
    "label": "Loan Category",
    "options": "Loan Category"
+  },
+  {
+   "fieldname": "co_lending_partner",
+   "fieldtype": "Link",
+   "label": "Co lending Partner",
```

Comment:
@tohidmalik002 the label and field name here should be loan partner and not co lending partner.